### DISCUSSION TITLE
Conta sinalizada

### DISCUSSION THREAD

**Discussion by RyanJrdev** [community/community#114252] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Pessoal, já faz alguns dias que minha conta foi sinalizada, por algum motivo que ainda não sei(Alguns meses atrás, fui hackeado, e o mesmo teve acesso ao meu GIThub, seria isso o motivo?), já enviei ticket para o Github porém não obtive retorno, alguém já passou por isso ou sabe como solucionar?
![Sinalizada](https://github.com/community/community/assets/142917990/d9a14d80-8d00-42c7-985b-7f523d0364b3)

**Comment by william-ks** [comment]
Eu também estou com o mesmo problema. você chegou a uma solução ?
